src/tooltip/tooltip.js

```diff
@@ -330,11 +339,17 @@ angular.module( 'ui.bootstrap.tooltip', [ 'ui.bootstrap.position', 'ui.bootstrap
 
               triggers = getTriggers( val );
 
-              if ( triggers.show === triggers.hide ) {
-                element.bind( triggers.show, toggleTooltipBind );
-              } else {
-                element.bind( triggers.show, showTooltipBind );
-                element.bind( triggers.hide, hideTooltipBind );
+              if (triggers.show !== 'manual') {
```

Comment:
Why is this necessary? As far as I can tell, there is no reference to `manual` at all - this if conditional should be removed.